src/main/kotlin/no/nav/soknad/arkivering/soknadsmottaker/Metrics.kt

```diff
@@ -0,0 +1,45 @@
+package no.nav.soknad.arkivering.soknadsmottaker
+
+import io.prometheus.client.Counter
+
+internal object Metrics {
+
+
+	private val SOKNAD_NAMESPACE = "soknadinnsending"
+	private val SUB_SYSTEM = "soknadsMottaker"
+	private val NAME = "innsendte"
+	private val HELP = "Number of applications and documents received and published to Kafka stream"
+	private val LABEL_NAME = "tema"
+	private val ERROR = "feil"
+	private val HELP_ERROR = "Number of applications and documents failed receive and publish to Kafka stream"
```

Comment:
Gramatikken er ikke helt riktig. Kunde vi bare forkorte ner til "Number of errors" rett og slett, eller ger det ikke mening nog?

Reply:
Fikset teksten på teksten.